src/domain/index.ts

```diff
@@ -9,17 +11,25 @@ export interface DomainServices {
    * Note service instance
    */
   noteService: NoteService,
+
+  /**
+   * Auth service instance
+   */
+  authService: AuthService,
 }
 
 /**
  * Initiate services
  *
  * @param repositories - repositories
+ * @param authConfig - auth config
  */
-export function init(repositories: Repositories): DomainServices {
+export function init(repositories: Repositories, authConfig: AuthConfig): DomainServices {
```

Comment:
maybe it's more scalable to pass a whole config?

Reply:
done